### DISCUSSION TITLE
Error detected while processing /.config/nvim/init.lua:

### DISCUSSION THREAD

**Discussion by jkemp814** [folke/snacks.nvim#114] [Q&A]
My NeoVim/LazyVim updated to new snacks.nvim plugin with the new dashboard. It seems to run fine without any errors. Just the basic dashboard, though. (meaning nothing but the 'LAZYVIM' logo, no shortcut keys displayed)

I forgot I have a script to update many things on my Fedora 41 Linux Silverblue. In that script, it would update all the plugins in LazyVim from the command line. After it ran that script, I now get this error below.

Here is the script to update the plugins:

`nvim --headless "+Lazy! sync" +qa`

Apparently that cannot be used anymore, or needs to be modified.

Is there some way to start over with all my plugins? Is the init.lua file corrupted at this point?
```
Error detected while processing /var/home/james/.config/nvim/init.lua:
Invalid spec module: `plugins.New lua File`
Expected a `table` of specs, but a `nil` was returned instead
```

Here is my init.lua file. Not much in it.

```
-- bootstrap lazy.nvim, LazyVim and your plugins
require("config.lazy")
```

**Comment by jkemp814** [comment]
Okay, I found out there was a new file name 'New lua File.lua' Don't know how it got there, but I deleted it and the error is gone. The original dashboard is back. 

How do I get to the new 'snacks' dashboard?

**Reply by IamSmeagol** [reply]
Can you show you neovim config? Are you on the latest version of LazyVim? Have you tried running `:checkhealth snacks`?
Also why not use the lazy plugin manager to update everything?

**Comment by jkemp814** [comment]
Yes, let me gather the Neovim config. I'm on the latest LavyVim. I have run :`checkhealth snacks` and all checks came back "OK." I have removed that section from my update script and from now on run the update from dashboard.

The keyboard shortcut items are now showing after a reboot. Initially it wasn't showing, so I thought I had to add it to a layout type file. I'm assuming all is well, now.

**Reply by IamSmeagol** [reply]
Are you seeing the dashboard now?
If not make sure it's enabled, all the snacks are off by default. However since you are using lazyvim the latest update should have turned it on.
